libr/cons/cons.c

```diff
@@ -367,6 +368,18 @@ R_API bool r_cons_enable_mouse(const bool enable) {
 #endif
 }
 
+// Stub function that cb_logfunc_print gets assigned to in logging.c by default
+// This allows Cutter to set per-task logging redirection
+R_API void r_cons_logging_stub(const char *output, const char *funcname, const char *filename,
+ unsigned int lineno, unsigned int level, const char *fmtstr, ...) {
+	if (!I.null && I.context->logging_callback) {
+		va_list args;
+		I.context->logging_callback (output, funcname, filename, lineno, level, fmtstr, args);
```

Comment:
can we have this logging callback as an RList? so we can hook many listeners? this can be used for having events that update the UI and keep different instances of r2 in sync

Reply:
> can we have this logging callback as an RList? so we can hook many listeners?

this seems useful to save both to screen and to file or other places

> this can be used for having events that update the UI and keep different instances of r2 in sync

what do you mean with this?

Reply:
passing args without va_start? this seems like UB